GhidraBuild/IDAPro/Python/7xx/python/idaxml.py

```diff
@@ -151,7 +150,10 @@ def display_version(self, what):
         Args:
             what: String indicating Exporter, Importer, or Loader 
         """
-        f = ida_diskio.idadir('python') + '/idaxml.py'
+        if os.path.isfile(os.path.join(ida_diskio.get_user_idadir(), 'python', 'idaxml.py')):
+            f = os.path.join(ida_diskio.get_user_idadir(), 'python', 'idaxml.py')
+        else:
+            f = os.path.join(ida_diskio.idadir('python'), '/idaxml.py')
```

Comment:
```suggestion
            f = os.path.join(ida_diskio.idadir('python'), 'idaxml.py')
```

Without this fix, the resulting path becomes `/idaxml.py` instead of `~/.idapro/idaxml.py` when using `os.path.join`.